orion_mcp.py

```
@@ -902,9 +1094,126 @@ def _timestamp_after(timestamp_val, cutoff_datetime: datetime) -> bool:
+    try:
+        prior_result = await run_orion(
+            config=full_path,
+            version=version,
+            lookback=str(lookback * 2),
+            input_vars=iv,
+        )
+        prior_sum_raw = await summarize_result(prior_result)
+        if isinstance(prior_sum_raw, dict):
+            prior_sum = prior_sum_raw
+    except Exception:
+        pass
```

Ruff (0.16.1)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)